lib/govuk_publishing_components/presenters/big_number_helper.rb

```diff
@@ -0,0 +1,33 @@
+module GovukPublishingComponents
+  module Presenters
+    class BigNumberHelper
+      def initialize(local_assigns)
+        @label = local_assigns[:label]
+        @href = local_assigns[:href]
+        @number = local_assigns[:number]
+      end
+
+      def value_classes
+        class_list = ["gem-c-big-number__value"]
+
+        if @label.nil? && @href
+          class_list << "gem-c-big-number__value--decorated"
+        end
+
+        class_list
+      end
+
+      # We amend the number here if it ends with a "+" to account for small symbols awkwardly sitting above the baseline in our typeface and adjust them to sit at the baseline for the purpose of visual harmony
+      # Currently the only small symbol used with big numbers are pluses so they are specifically targetted in the below code
+      def formatted_number_string
+        if @number
```

Comment:
Wondering whether if checking if this exists could be removed for setting a default instead of an empty string in the initializer instead?

Reply:
Also ELI5 this please

Reply:
sorry for confusing you with my instead instead